commit a5193b0eab8cb02452a3a0093800cde4322c104b (refs/changes/12/2512/4)
Author: Brittany Wald <brittany.wald@joyent.com>
Date:   2017-09-20T16:22:44+00:00 (2 years, 1 month ago)
    
    MANTA-3413 node-fash seems to intend add-data behavior it does not enable
    Reviewed by: Tim Kordas <tim.kordas@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/bin/fash.js b/bin/fash.js
index ea04d9c..607cc17 100755
--- a/bin/fash.js
+++ b/bin/fash.js
@@ -210,7 +210,7 @@ Fash.prototype.do_deserialize_ring.help = (
 Fash.prototype.do_add_data = function (subcmd, opts, args, callback) {
     var self = this;
 
-    if (opts.help || args.length !== 0 || !opts.v || !opts.d || !opts.b) {
+    if (opts.help || args.length !== 0 || !opts.v || !opts.b) {
         this.do_help('help', {}, [subcmd], function (err) {
             return callback(err ? err : true);
         });
diff --git a/package.json b/package.json
index 8935c96..4f08587 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "dynamo",
     "hash ring"
   ],
-  "version": "2.4.0",
+  "version": "2.4.1",
   "repository": {
     "type": "git",
     "url": "https://github.com/yunong/node-fash.git"
